scripts/tfjs-yolo/pip.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I think we should not upload pipeline file.

Reply:
done